extension/content/customPersonaEditor.js

```diff
@@ -263,6 +263,9 @@ var CustomPersonaEditor = {
       case PersonaService.FIREFOX_ID:
  	window.close();
         break;
+      case PersonaService.SEAMONKEY_ID:
+    window.close();
```

Comment:
I won't get into this discussion again. Please properly format new code you write. We can take care of the rest later.